packages/firestore/src/platform/browser_lite/fetch_connection.ts

```diff
@@ -28,17 +27,6 @@ import { StringMap } from '../../util/types';
  * (e.g. `fetch` or a polyfill).
  */
 export class FetchConnection extends RestConnection {
-  /**
-   * @param databaseInfo - The connection info.
-   * @param fetchImpl - `fetch` or a Polyfill that implements the fetch API.
-   */
-  constructor(
-    databaseInfo: DatabaseInfo,
-    private readonly fetchImpl: typeof fetch
-  ) {
-    super(databaseInfo);
```

Comment:
This consructor is removed because it would now have the same signature as RestConnection's constructor, so we just use that constructor instead.